### ISSUE TITLE
Stickers not big enough

### ISSUE THREAD

**Issue by txomon** [zhukov/webogram#1175]
When going to choose stickers, I can't see them at all. They are too tiny! However, if I click in one to add them to my collections, they are perfect size to choose the one I want.

**Comment by migueloop** [comment]
To preview stickers hold your click so a bigger preview appears.
Does it solve your problem?

**Comment by zhukov** [comment]
@migueloop We don't have long click for preview in web. but you can click on pack title to open a pack in modal window with bigger size thumbs.

**Comment by txomon** [comment]
Not really, because I would need to go one by one, I need to watch out for
details in each of them... Maybe putting it big on hover is good enough,
but it needs to be big

**Comment by txomon** [comment]
I have no idea on how to open a pack and then insert from there... also I
would love to be able to see all packs together, as I am more interested in
the content than the origin

**Comment by zhukov** [comment]
hmm... you can easily click on any sticker in pack to send it:
![screen shot 2016-06-28 at 3 10 44 pm](https://cloud.githubusercontent.com/assets/775173/16414816/8f074a3a-3d42-11e6-949d-dd323c628a03.png)

**Comment by txomon** [comment]
Yeah, but pack by pack, and many times you just have packs with a few... The ideal would be browsing

**Comment by pumano** [comment]
I vote for this feature too. Because it's not trivial to read text in a small preview window.

I also publish similar feature request to macOS client: https://github.com/overtake/telegram/issues/344

**Comment by pootzko** [comment]
It would be great if sticker browsing was opened in that big modal by default and if you could go through all packs in it, not set-by-set (that's a few extra unnecessary steps).
